src/Sly/NotificationPusher/Model/GcmMessage.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Sly\NotificationPusher\Model;
+
+
+class GcmMessage extends Message
+{
+    /**
```

Comment:
I think we need to have the ability to set the data key here as well otherwise this is not that useful.

Reply:
@morloderex hey, it extends Message. So you could use it like a regular Message with data or set some notificationData in addition.

Reply:
Yeah but having both in the same class makes a lot of sense to me.

Reply:
@morloderex personally I agree, that having `private $data;`
would be better
just because in the core design Message class initially use arrays, so it would require more changes and BC or support both ways.
what if a user set data via field and via array param, then they need to be merged. but in which order? not trivial.

Reply:
better add as separated PR